main/sigmabot.py

```diff
@@ -181,6 +181,11 @@ async def difficulty(ctx):
 async def seed(ctx):
 	await ctx.send(fq.faq_seed())
 
+## FORUM
+@client.command()
+async def seed(ctx):
```

Comment:
@cafebedouin this makes it so the bot command is called `.seed` and calls the `faq_forum()` command